packages/now-cli/src/commands/domains/inspect.ts

```diff
@@ -145,37 +154,50 @@ export default async function inspect(
   );
   output.print('\n');
 
-  output.print(chalk.bold('  Verification Record\n\n'));
-  output.print(
-    `${dnsTable([['_now', 'TXT', domain.verificationRecord]], {
-      extraSpace: '    ',
-    })}\n`
-  );
-  output.print('\n');
-
   if (!domain.verified) {
-    output.warn(`This domain is not verified. To verify it you should either:`);
+    output.warn(`This domain is not verified. `);
     output.print(
-      `  ${chalk.gray(
-        'a)'
-      )} Change your domain nameservers to the intended set detailed above. ${chalk.gray(
+      `  To verify it you should, change your domain nameservers to the intended set detailed above. ${chalk.gray(
         '[recommended]'
       )}\n`
     );
-    output.print(
-      `  ${chalk.gray(
-        'b)'
-      )} Add a DNS TXT record with the name and value shown above.\n\n`
-    );
     output.print(
       `  We will run a verification for you and you will receive an email upon completion.\n`
     );
+    output.print('  Read more: https://err.sh/now/domain-verification\n\n');
```

Comment:
Let's move this document to the [FAQ](https://vercel.com/docs/v2/platform/frequently-asked-questions) like the newer error messages and then use `vercel.link` shortener instead.

Reply:
@styfle we already have [this](https://vercel.com/docs/v2/custom-domains#step-4:-configuring-the-domain), which is similar, do you think we should use this instead?

Reply:
That works 👍